staff/ana-sanjuan/cv/index.html

```diff
@@ -5,121 +5,115 @@
+                <address>
+                    <p>Email: <a href= "mailto:lorem@lorem.com">lorem@lorem.com</a></p>
+                    <p>Address: Lorem Ipsum Street 1, Barcelona<br></p>
+                    <p>Phone: +00-5555-555</p>   
+                </address>
+                <nav>
+                    <a href="https://www.wikipedia.org/">github</a> |
+                    <a href="https://www.wikipedia.org/">LinkedIn</a> |
+                    <a href="https://www.wikipedia.org/">Skype</a> |
+                </nav>
+             </div>
+        </section>
 
-    <hr>
+        <div class='first-row container'>
+            <section class='skills section'>
+                <h2 class='section__title'>Skills</h2>
+                <ul class='section__ulist'>
+                    <li><b>Programming Languages:</b> lorem, lorem and lorem.</li>
+                    <li><b>Databases:</b> lorem, lorem, lorem and lorem.</li>
+                    <li><b>Platforms:</b> lorem, lorem, lorem and lorem.</li>
+                    <li><b>Version Control:</b> lorem, lorem, lorem and lorem.</li>
+                    <li><b>Operating Systems:</b> lorem, lorem, lorem and lorem.</li>
+                    <li><b>Other Tools:</b> lorem, lorem, lorem and lorem.</li>
+                    <li><b>Soft Skills:</b> lorem, lorem, lorem and lorem.</li>
+                </ul>
+            </section>
+           
+            <section class='experience section' >
+                <h2 class='section__title'>Work Experience</h2>
+                <div class='div__item'>
+                    2010/14 -The Company(BCN)  -Full Stack 
+                </div>
+                <div class='div__item'>
+                    2014/17 -The Other(LDN) -Full Stack
+                </div>
+                <div class='div__item'>
+                    2017/18 -One More(LA) -Full Stack
+                </div>
+            </section>
+        </div>
+        <div class='nd-row container'>
+            <section class='education section'>
+                <h2 class='section__title'>Education</h2>
+                <div class='div__item'>
+                    2000/05 -Life(BCN) -Full Lorem 
+                </div>
+                <div class='div__item'>
+                    2005/10 -Life(LDN) -Full Ipsum
+                </div>
+ 
+            </section>
 
-    <section>
-        <h2>References</h2>
-        <p>Available under request</p>
-    </section>
+            <section class='hobbies section'>
+                <h2 class='section__title'>Hobbies</h2>
+                
+                <ul class='section__ulist star'>
+                    <li>Lindy Hop 
+                    <span class="fa fa-star checked"></span>
+                    <span class="fa fa-star checked"></span>
+                    <span class="fa fa-star checked"></span>
+                    <span class="fa fa-star"></span>
+                    <span class="fa fa-star"></span></li>
```

Comment:
in this case, why do you use <span>? the classes used in the span are not in the css file.

Reply:
fa and fa-star are the clasess that define the star per se, and checked is the filler.
fa and fa-star are taken from the link("https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"), and checked is in the css